src/main/java/macrobase/conf/MacroBaseConf.java

```diff
@@ -166,6 +170,9 @@ public BatchTrainScore constructTransform (TransformType transformType)
             case BAYESIAN_NORMAL:
                 log.info("Using Bayesian Normal transform.");
                 return new BayesianNormalDensity(this);
+            case GAUSSIAN_MIXTURE_EM:
```

Comment:
i put `EM` in the end to indicate it uses EM algorithm, but it seems to be too subtle.
`GAUSSIAN_MIXTURE_EM` (This is finite mixture using EM)
`VARIATIONAL_GAUSSIAN_MIXTURE` (This is finite mixture using Bayesian Variational Inference)
`DP_GAUSSIAN_MIXTURE` (This is the inifinite mixture, using Bayesian Variational Inference)